packages/hash/frontend/src/pages/[account-slug]/index.page.tsx

```diff
@@ -46,21 +48,43 @@ const Page: NextPageWithLayout = () => {
     );
   }
 
-  const workspaceName = ownWorkspace ? "your" : `${thisOrg!.name}'s`;
-
   return (
     <>
-      <Box component="header" mt={1.5}>
-        <Typography mb={2} variant="h1">
-          Hi, {user.properties.preferredName}!
+      <Typography mb={3} variant="h2">
+        Welcome to HASH
+      </Typography>
+      <Box maxWidth="75ch">
+        <Typography mb={3}>
+          HASH is an open-source, data-centric, all-in-one workspace built atop
+          the open <Link href="https://blockprotocol.org">Block Protocol</Link>.
         </Typography>
-        <Typography mb={2} variant="h3">
-          Welcome to {workspaceName} workspace.
+        <Typography mb={3}>
+          <strong>
+            <FontAwesomeIcon
+              icon={faWarning}
+              sx={({ palette }) => ({
+                color: palette.orange[50],
+                mr: 0.5,
+              })}
+            />{" "}
+            HASH is not ready for production use.
+          </strong>{" "}
+          It is not secure or optimized and is missing key features. Please
+          visit the{" "}
+          <Link href="https://github.com/hashintel/hash/tree/main/packages/hash">
+            GitHub repository
+          </Link>{" "}
+          for the latest updates, or learn about the long-term{" "}
+          <Link href="https://hash.ai">here.</Link>
```

Comment:
Might be worth aligning `<a>link</a>.` and `<a>link.</a>`

- `<Link href="https://blockprotocol.org">Block Protocol</Link>.`
- `<Link href="https://hash.ai">here.</Link>`
- `<Link href="https://github.com/hashintel/hash/tree/main/packages/hash#integration-with-the-block-protocol">read the documentation to get started.</Link>`

I’m in favor of `<a>link</a>.` but happy with any option as long as it’s consistent.